### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by kevmoo**
This removes all hints from generated Dart code in the 'lib' dir

Will do a follow-up to cleanup test

### CODE REVIEW

**Comment by kevmoo** [comment]
@mhevery I don't have merge access. Feel free.

File: modules/angular2/src/core/application.js [hunk 1]
```diff
@@ -7,9 +8,8 @@ import {Reflector, reflector} from 'angular2/src/reflection/reflection';
 import {Parser, Lexer, ChangeDetection, dynamicChangeDetection, jitChangeDetection} from 'angular2/change_detection';
 import {TemplateLoader} from './compiler/template_loader';
 import {DirectiveMetadataReader} from './compiler/directive_metadata_reader';
-import {DirectiveMetadata} from './compiler/directive_metadata';
 import {List, ListWrapper} from 'angular2/src/facade/collection';
-import {PromiseWrapper} from 'angular2/src/facade/async';
+import {Future, PromiseWrapper} from 'angular2/src/facade/async';
```
Comment (vicb) [inline modules/angular2/src/core/application.js]: Is `Future` defined in JS ?

**Comment by kevmoo** [comment]
@mhevery ping – I need someone to merge this in for me

File: modules/angular2/src/core/application.js (hunk 1 above)
Comment (kevmoo) [inline modules/angular2/src/core/application.js]: Nope. This should be Promise. Will fix.

**Comment by kevmoo** [comment]
Rebased & fixed

**Pull request closed without merging**